src/Halogen/Components/Select/Rotary/Component.purs

```diff
@@ -0,0 +1,348 @@
+module Halogen.Components.Select.Rotary.Component
```

Comment:
Sounds good. Maybe even `SlamData.Halogen.Select` or `SlamData.Component.Select`? Any of the above is fine by me :+1:

I'm still reviewing the rest, it's taking a while :smile: